web/src/locales/zh/data.json

```diff
@@ -129,7 +129,8 @@
     "No account yet?": "没有账号？",
     "sign up now": "立即注册",
     "To access": "访问",
-    "Sign in with {type}": "{type}登录"
+    "Sign in with {type}": "{type}登录",
+    "Logged in successfully": "登录成功"
```

Comment:
Don't we have this before?

Reply:
Yes. All "Logged in successfully" message did not use i18n.
![$ C3B$~2ZE)B %BJK0 W@09](https://user-images.githubusercontent.com/51157931/125435268-d0e8308b-67de-4461-a78a-e00bba6bd12f.png)